mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulHome.kt

```diff
@@ -85,12 +88,19 @@ fun StatefulHome(
         StatefulSettingsScreen(
             user = user,
             onEditProfileNameClick = { controller.navigate(SettingEditProfileNameRoute) },
+            onEditProfileImageClick = { controller.navigate(SettingEditProfileImageRoute) },
             modifier = Modifier.fillMaxSize(),
             contentPadding = paddingValues)
       }
       dialog(SettingEditProfileNameRoute) {
         StatefulEditProfileNameDialog(user = user, onClose = { controller.popBackStack() })
       }
+      dialog(SettingEditProfileImageRoute) {
+        StatefulEditProfileImageDialog(
+            user = user,
+            onSave = { controller.popBackStack() },
+            onCancel = { controller.popBackStack() })
```

Comment:
Is the behaviour different across the `onSave` and `onCancel` callbacks ? We probably should only have a single `onClose` function, exactly like for `StatefulEditProfileNameDialog`.